### ISSUE TITLE
Bootloader won't start

### ISSUE THREAD

**Issue by sc16me** [Ralim/IronOS-dfu#20]
Yesterday my iron began to lower the temperature by 200 degrees, I tried to update the firmware, but it did not work. I also tried doing a full reset. I installed the stock firmware, the problem disappeared, but I wanted to use the IronOS. I tried cleaning the board with alcohol, but that didn't help either. I have v2.51A PCB revision. I tried to flash runtime, but there was a black screen. DFU 3.45. I decided to connect via st-link. I accidentally erased the firmware without saving the backup (applause). Next, I flashed the bootloader and firmware (hex files). Now the soldering iron behaves strangely - when power is applied, nothing happens, the screen is black, the computer does not see the connected devices. Through the st-link I can start the iron, to do this I click connect, then disconnect and the firmware is loaded! But without such manipulations, simply by supplying power, it is not possible to achieve a start. Plz help

**Comment by Ralim** [comment]
Hello,

To start with can you confirm what device you have, and also if the main mcu is a GD32 clone or an STM32 original ic?

**Comment by sc16me** [comment]
> Hello,
> 
> To start with can you confirm what device you have, and also if the main mcu is a GD32 clone or an STM32 original ic?

![IMG_5553](https://github.com/Ralim/IronOS-dfu/assets/1618809/01e22f22-9ead-41e6-ba24-27954f94b50c)
![IMG_5554](https://github.com/Ralim/IronOS-dfu/assets/1618809/18015785-2b65-4ae3-a3bb-158879e4cfed)
![IMG_5555](https://github.com/Ralim/IronOS-dfu/assets/1618809/a90cf918-b530-43c9-b3e7-89e514962dc6)
![IMG_5556](https://github.com/Ralim/IronOS-dfu/assets/1618809/02bf8e32-0f95-45b3-99c6-fd75c3dce3a2)

**Comment by sc16me** [comment]
Apparently the problem is not in the software. After some time, the soldering iron started to start, but it became impossible to get into dfu. It also showed 200C when cold. I decided to solder to the pins for stlink, however stlink only saw an unknown device. I reassembled the soldering iron, but it stopped turning on completely. No dfu, no firmware, nothing. Well, perhaps we should let go of our old friend. I bought Pinecil v1
